tests/bluetooth/shell/src/main.c

```diff
@@ -104,11 +100,9 @@ SHELL_CMD_REGISTER(hrs, &hrs_cmds, "Heart Rate Service shell commands",
 
 void main(void)
 {
-	(void)shell_init(&uart_shell, NULL, true, true, LOG_LEVEL_INF);
-
-	print(&uart_shell, "Type \"help\" for supported commands.");
-	print(&uart_shell, "Before any Bluetooth commands you must `bt init`"
-	      " to initialize the stack.");
+	printk("Type \"help\" for supported commands.");
```

Comment:
Why did you change this from `print` to `printk`?

Reply:
shell_fprintf must be used only in the scope of the command.

Reply:
`print` is a macro, in fact it is a function `shell_fprintf`. This function **shall not** be used outside of command context what is a case here. I was trying to print this welcome message by Logger.

Reply:
What is more you do not have access to uart_shell any more because it is created in shell_uart.c file.

Reply:
OK, that makes sense.